Formula/range-v3.rb

```diff
@@ -0,0 +1,33 @@
+class RangeV3 < Formula
+  desc "Experimental range library for C++11/14/17"
+  homepage "https://ericniebler.github.io/range-v3/"
+  url "https://github.com/ericniebler/range-v3/archive/0.3.6.tar.gz"
+  sha256 "ce6e80c6b018ca0e03df8c54a34e1fd04282ac1b068cd39e902e2e5201ac117f"
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".",
+      "-DRANGE_V3_TESTS=OFF",
```

Comment:
`RANGE_V3_HEADER_CHECKS` and `RANGE_V3_TESTS` appear to be off by default, already.

Reply:
Indentation should left-align on the second cmake argument